plugin-api/beta/client/components/index.js

```diff
@@ -1 +1 @@
-export {Slot, Copy} from 'coral-framework/components';
\ No newline at end of file
+export {Slot, Copy} from 'coral-framework/components';
```

Comment:
What I meant was, that there is no component `Copy` in `coral-framework/components`, what are you trying to export here?